2017-10-14 15:34:21,761 handlers.py[line:104] ERROR Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/sanic/server.py", line 151, in data_received
    self.parser.feed_data(data)
  File "httptools/parser/parser.pyx", line 171, in httptools.parser.parser.HttpParser.feed_data (httptools/parser/parser.c:2721)
httptools.parser.errors.HttpParserInvalidMethodError: invalid HTTP method

2017-10-14 15:34:21,762 server.py[line:298] INFO 
2017-10-14 17:27:27,990 server.py[line:206] INFO 
2017-10-15 01:13:56,606 handlers.py[line:104] ERROR Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/sanic/server.py", line 151, in data_received
    self.parser.feed_data(data)
  File "httptools/parser/parser.pyx", line 171, in httptools.parser.parser.HttpParser.feed_data (httptools/parser/parser.c:2721)
httptools.parser.errors.HttpParserInvalidMethodError: invalid HTTP method

2017-10-15 01:13:56,607 server.py[line:298] INFO 
2017-10-15 03:27:56,085 server.py[line:206] INFO 
2017-10-15 05:00:42,418 handlers.py[line:104] ERROR Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/sanic/server.py", line 151, in data_received
    self.parser.feed_data(data)
  File "httptools/parser/parser.pyx", line 171, in httptools.parser.parser.HttpParser.feed_data (httptools/parser/parser.c:2721)
httptools.parser.errors.HttpParserInvalidMethodError: invalid HTTP method

2017-10-15 05:00:42,419 server.py[line:298] INFO 
2017-10-15 06:11:35,548 handlers.py[line:104] ERROR Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/sanic/server.py", line 151, in data_received
    self.parser.feed_data(data)
  File "httptools/parser/parser.pyx", line 171, in httptools.parser.parser.HttpParser.feed_data (httptools/parser/parser.c:2721)
httptools.parser.errors.HttpParserInvalidMethodError: invalid HTTP method

2017-10-15 06:11:35,549 server.py[line:298] INFO 
2017-10-15 10:56:43,719 handlers.py[line:104] ERROR Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/sanic/server.py", line 151, in data_received
    self.parser.feed_data(data)
  File "httptools/parser/parser.pyx", line 171, in httptools.parser.parser.HttpParser.feed_data (httptools/parser/parser.c:2721)
httptools.parser.errors.HttpParserInvalidMethodError: invalid HTTP method

2017-10-15 10:56:43,719 server.py[line:298] INFO 
